code/modules/clothing/head/armor.dm

```diff
@@ -504,6 +504,8 @@
 	icon_state = "crusader_hemet"
 	item_state = "crusader_hemet"
 	flags_inv = HIDEEARS|HIDEEYES|BLOCKHAIR
+	tint = TINT_LOW
```

Comment:
hidden balance change

Reply:
Good catch, seems I forgot to put it in the description 👍